spec/services/state_file/after_transition_messaging_service_spec.rb

```diff
@@ -115,20 +118,11 @@
   describe "#send_efile_submission_terminal_rejected_message" do
     let(:message) { StateFile::AutomatedMessage::TerminalRejected }
 
-    it "sends the terminal rejected refund email" do
+    it "sends the terminal rejected refund email and text" do
       expect do
         messaging_service.send_efile_submission_terminal_rejected_message
       end.to change(StateFileNotificationEmail, :count).by(1)
-
-      expect(efile_submission.message_tracker).to include "messages.state_file.terminal_rejected"
-      expect(StateFile::MessagingService).to have_received(:new).with(intake: intake, submission: efile_submission, message: message, body_args: body_args)
-    end
-
-    it "sends the terminal rejected refund sms" do
-      expect do
-        messaging_service.send_efile_submission_terminal_rejected_message
-        perform_enqueued_jobs
-      end.to change(FakeTwilioClient.messages, :count).by(1)
```

Comment:
since the actual twilio-message-sending is tested for the `StateFileNotificationTextMessage#after_create_commit` hook (and the job that it kicks off), it seems unnecessary to use `MockTwilio` here and should be sufficient to check that the `StateFileNotificationTextMessage` was created